_config.yml

```diff
@@ -16,9 +16,16 @@ sass:
 host:           localhost # same as default value '127.0.0.1' but reads better
 
 # Links
-main:           "https://getbootstrap.com"
-expo:           "https://expo.getbootstrap.com"
-themes:         "https://themes.getbootstrap.com"
+current_version: "4.3.1"
+docs_version:    "4.3"
+
+main:            "https://getbootstrap.com/"
+repo:            "https://github.com/twbs/bootstrap"
+slack:           "https://bootstrap-slack.herokuapp.com"
```

Comment:
@mdo this reminds me, what are we going to do with Slack? Can I help maybe? Otherwise we need to fix the link somehow

Reply:
I'm not sure, I couldn't get the slackin repo up and running correctly anywhere. If you could take a look at that project and see if you can run it, that'd be awesome.